### PR TITLE
created and styled feature card component

### PR DESCRIPTION
**Pull request by hamees-sayed**

### CODE REVIEW

File: assets/scss/_styles_project.scss [hunk 1]
```diff
@@ -401,6 +401,64 @@ section:nth-of-type(2) {
...
+/* Feature Card Component */
+.feature-container {
+  padding: 0 !important;
+  background-image: url(../semi_circle.svg);
+  background-position: top right;
+  background-repeat: no-repeat;
+}
+
+.feature-text-container h1 {
+  font-size: 3rem;
```
Comment (thisisobate) [inline assets/scss/_styles_project.scss:414]: `3rem` looks a bit too much. Try reducing the size to fit with the rest of the document.

File: assets/scss/_styles_project.scss [hunk 2]
```diff
@@ -401,6 +401,64 @@ section:nth-of-type(2) {
...
+  font-weight:$font-weight-bold;
+}
+
+.feature-text-container p {
+  font-size: 2.25rem;
```
Comment (thisisobate) [inline assets/scss/_styles_project.scss:419]: Same here. Reduce the size to fit with the rest of the document

File: assets/scss/_styles_project.scss [hunk 3]
```diff
@@ -401,6 +401,64 @@ section:nth-of-type(2) {
...
+.order-number {
+  position: absolute;
+  font-size: 9rem;
+  color: rgba(26, 26, 26, 0.1);
+  margin-left: 0.05em;
+  font-weight: $font-weight-bold;
+}
+
+.feature-title h1 {
+  font-size: 2rem;
```
Comment (thisisobate) [inline assets/scss/_styles_project.scss:457]: same here...reduce size

File: content/en/_index.html [hunk 4]
```diff
@@ -101,6 +101,79 @@ <h4 class="title is-5">Auditing and Compliance</h4>
   </div>
 </section>
 
+{{% blocks/lead color="white" %}}
+<section class="feature-container">
+  <div class="feature-text-container text-center">
+    <h1 lass="feature-card-heading">What makes Notary unique</h1>
+    <p class="feature-card-caption">Lorem Ipsum is simply dummy text of the printing and <br/> typesetting industry. Lorem Ipsum has been the industry's standard<br/>
+    dummy text ever since the 1500s, when an unknown printer took a galley</p>
+  </div>
```
Comment (thisisobate) [inline content/en/_index.html:110]: Add padding around this section. You can take a look at how it's done in the `adopted and trusted` section: [https://github.com/thisisobate/notaryproject.dev/blob/bb629d8417c82e4c4b2ba5a1963d8f362a244b90/content/en/_index.html#L107](https://github.com/thisisobate/notaryproject.dev/blob/bb629d8417c82e4c4b2ba5a1963d8f362a244b90/content/en/_index.html#L107)

**Review by thisisobate — CHANGES_REQUESTED** [review]
Great work! @hamees-sayed 
I left a few suggestions for you to consider. Let me know when you address them and I'll approve right away!

File: assets/scss/_styles_project.scss (hunk 3 above)
Comment (hamees-sayed) [reply assets/scss/_styles_project.scss:457]: Is it okay if I also resize the entire box? 
image

**Review by thisisobate — APPROVED** [review]
LGTM! Thanks a bunch! @hamees-sayed

**Pull request merged**